queries/puppet/highlights.scm

```diff
@@ -0,0 +1,194 @@
+; Variables
+
+(identifier) @variable
+
+; Includes
+
+"include" @include
+
+(include_statement (identifier) @type)
+
+(include_statement (class_identifier (identifier) @type . ))
+
+; Keywords
+
+[
+  "class"
+  "inherits"
+  "node"
+  "type"
+  "tag"
+] @keyword
+
+[
+  "define"
+  "function"
+] @keyword.function
+
+[
+  "if"
+  "elsif"
+  "else"
+  "case"
+] @conditional
+
+(default_case "default" @conditional)
```

Comment:
wouldn't default be `@label` in C/C++/Java/Dart?

Reply:
Hmm, I don't particularly like that, it would fit better as being its own "case", i.e. a condition imo